buttercup.el

```diff
@@ -1701,5 +1701,10 @@ With buttercup minor mode active the following is activated:
       (cl-dolist (form imenu-forms)
         (setq imenu-generic-expression (delete form imenu-generic-expression))))))
 
+;; Local Variables:
+;; indent-tabs-mode: nil
+;; sentence-end-double-space: nil
+;; End:
 (provide 'buttercup)
 ;;; buttercup.el ends here
+
```

Comment:
At the risk of sounding highly pedantic, but there should not be an empty line at the end of the file, either :-D

Reply:
I have no problem complying with that, but I am a bit surprised. Where is that policy stated?

Reply:
Sorry, I see what you mean now. I'll push an update.

Reply:
Updated now. Rebased the branch as well.